Source/TemplateLib/CommonSqlCode.cs

```diff
@@ -6233,15 +6245,16 @@ public string GetVisualStudioProductVersionString( VSNetVersion version )
 
         public string GetVisualStudioToolVersionString( VSNetVersion version )
         {
-            string toolsVersion = "3.5";
+            string toolsVersion = "Current";

```

Comment:
Is there a reason we are changing this to "Current" rather than a version number?  Current usually does not age well if the code does not consistently add new versions.

Reply:
This is because the newer VisualStudio project templates auto update from v2019 templates so we don't need to add support for newer versions in the future. But I have limited knowledge so feel free to change this if it doesn't make sense.